installer/helper.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+
+# Copyright 2020 The SODA Authors.
+#   
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os, sys, shutil
+import logging
+from logging.handlers import RotatingFileHandler
+from datetime import datetime
+
+log_filename = 'sim_installer.log' + datetime.now().strftime("%d_%m_%Y_%H_%M_%s")
+LOGGING_FORMAT = "[%(asctime)s] [%(levelname)s] [%(filename)s] " \
+    "[%(funcName)s():%(lineno)s] [PID:%(process)d TID:%(thread)d] %(message)s"
+LOGGING_LEVEL = "INFO"
+logger = None
+logfile=''
+sim_log_dir = '/var/log/sodafoundation/'
+
+def init_logging():
+    global logfile
+    global logger
+    
+    try:
+        os.mkdir(sim_log_dir)
+    except OSError as ose:
+        pass
```

Comment:
Please add an error message to report to user, about failure.

Reply:
As this is logger initialization, cannot log it..printing on stdout will not be good as it's just handling that log dir already exists..so just passing